index.html

```diff
@@ -94,90 +94,13 @@ <h1 class="greeting-text">Hey there. I’m <br> Abdul Ali.</h1>
 <!-- ----------------------------------------Recent work Setion------------------------------------------- -->
 
             <section class="recent-work grid" id="recent-work">
+                
```

Comment:
- [ ] Please it would be better to change the overflow of the popup here you can see that the user can scroll all over the background of the popup but not the popup itself he can't get a full view of the popup.
![image](https://user-images.githubusercontent.com/74744787/217950115-73a1964e-01be-47aa-aa29-6c7318df6e6e.png)